frontend/src/pages/PetProfile/PetProfileAddition/PetProfileNameAddition.tsx

```diff
@@ -28,14 +28,20 @@ const PetProfileNameAddition = () => {
         minLength={1}
         placeholder="여기를 눌러 아이의 이름을 입력해주세요."
         maxLength={10}
-        isValid={isValidInput}
+        value={petProfile.name}
+        isValid={isFirstRendered || isValidInput}
         onChange={onChangeName}
         design="underline"
         fontSize="1.3rem"
       />
       <ErrorCaption>
-        {isValidInput ? '' : '아이의 이름은 1~10글자 사이의 한글, 영어, 숫자만 입력 가능합니다.'}
+        {isFirstRendered || isValidInput
+          ? ''
+          : '아이의 이름은 1~10글자 사이의 한글, 영어, 숫자만 입력 가능합니다.'}
```

Comment:
```suggestion
        {(isFirstRendered || isValidInput) &&
          '아이의 이름은 1~10글자 사이의 한글, 영어, 숫자만 입력 가능합니다.'}
```
string만 렌더링되니까 이렇게 변경해도 좋을 것 같아요!

Reply:
```tsx
      <ErrorCaption>
        {!isFirstRenderedOrValidInput &&
          '아이의 이름은 1~10글자 사이의 한글, 영어, 숫자만 입력 가능합니다.'}
      </ErrorCaption>
```
요럭게 수정했습니다!